network/eni/eni.go

```diff
@@ -103,8 +103,8 @@ func (eni *ENI) DetachFromLink() error {
 	return nil
 }
 
-// getInterfaceByMACAddress returns the interface with the specified MAC address.
-func getInterfaceByMACAddress(macAddress net.HardwareAddr, interfaces []net.Interface) *net.Interface {
+// GetInterfaceByMACAddress returns the interface with the specified MAC address.
```

Comment:
I would just copy this function to your test code instead of exposing it from the ENI package and making it part of the contract.

Reply:
Sure. I will make that change.

Reply:
Updated in [ed8da834](https://github.com/aws/amazon-vpc-cni-plugins/pull/103/commits/ed8da834c06a5606b85bd5dc8ef6b8e93323c9fc).